test/generis.conf.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2002-2008 (original work) Public Research Centre Henri Tudor & University of Luxembourg (under the project TAO & TAO2);
+ *               2008-2010 (update and modification) Deutsche Institut für Internationale Pädagogische Forschung (under the project TAO-TRANSFER);
+ *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
+ *
+ */
+
+
+/**
+ * Generis Configuration used only for php unit tests.
+ *
+ * @author CRP Henri Tudor - TAO Team - {@link http://www.tao.lu}
+ * @package generis
+
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
+
+# local namespace
+define('LOCAL_NAMESPACE', '');
+
+# platform identification
+define('GENERIS_INSTANCE_NAME', '');
+define('GENERIS_SESSION_NAME', '');
+
+# paths
+define('ROOT_PATH', '');
```

Comment:
I think the ROOT_PATH could be predefined as `__DIR__  . DIRECTORY_SEPARATOR .  '..' . DIRECTORY_SEPARATOR .  '..' . DIRECTORY_SEPARATOR`;

I am afraid some test would be loading something from this path which will no longer exit anymore. I am not sure about this too. Wdyt?

Reply:
It would be very wrong if any unit test used this constant. So far I haven't seen any sign of it luckily. 

It just occurred to me now: what if I remove (comment) those ones which we know for sure doesn't used. To embrace the vision, that is for legacy only and for new tests proper DI must be in place.

WDYT?

Reply:
It would be very wrong if any unit test did depend on this constant. Luckily I haven't found any.

It just occurred to me: Is it better if I just remove all the "placeholder" constants (with no proper value) to embrace our vision that this is for legacy only?

WDYT?

Reply:
Yes, that is why I was not sure what I suggested. Thanks for the reply. I also think I cannot calculate right know if it will break or not some of the extensions we have. I think with time we will discover. It could be empty then.

Reply:
@gabrielfs7 thanks for sharing your thoughts. 

WDYT on having all the "empty" constants removed from this config? For instance `define('DEBUG_MODE', false);` would stay as it has proper value set, but `define('ROOT_URL', '');` would be gone as it has an obviously invalid (empty) value. Same filter to be applied on all constants. At then end we would end up having a few lines long config containing only relevant entries which is in addition also aligned with our "vision".